docs/reference/outlook.md

```diff
@@ -0,0 +1,140 @@
+# Outlook Connector
+
+The [Elastic Outlook connector](../connectors/sources/outlook.py) is built with the Elastic connectors Python framework and is available as a self-managed [connector client](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+## Availability and prerequisites
+
+This connector is available as a **connector client** from the **Python connectors framework**. To use this connector, satisfy all [connector client requirements](https://www.elastic.co/guide/en/enterprise-search/master/build-connector.html).
+
+This connector is in **beta** and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Usage
+
+To use this connector as a **connector client**, use the **Outlook** tile from the connectors list OR **Customized connector** workflow.
+
+For additional operations, see [Usage](https://www.elastic.co/guide/en/enterprise-search/master/connectors-usage.html).
+
+## Connecting to Outlook
+
+### Connect to Exchange Server
+
+To order to connect to Outlook, the connector fetches Active Directory users with the help of `ldap3` python library.
+
+### Connect to Office365 Outlook
+
+To order to connect to Outlook, user needs to create an Azure application as follows:
+
+1. Log into Microsoft [Azure Portal](https://portal.azure.com) and click on App registrations and register an application.
+2. From Overview tab, keep `Client ID` and `Tenant ID` handy.
+3. Go to **Certificates & secrets** and create a new client secret.
+4. Before configuring `Client ID`, `Tenant ID` and `Client Secret` in elastic, from API permissions tab - add below permission to fetch the Outlook content,
+    - Add permissions -> APIs my organization uses -> Office 365 Exchange Online -> full_access_as_app (Application permission)
+5. Now, user can configure `Client ID`, `Tenant ID` and `Client Secret` values to Outlook connector.
```

Comment:
Is this part of the Oauth App creation process, if not, consider removing.

Reply:
Could add a step here that says "Make note of `Client ID`, `Tenant ID` and `Client Secret`" and consider removing step 2 to avoid redundancy

Reply:
Example edit:


To integrate with the Outlook connector using Azure, follow these steps to create and configure an Azure application:

1. Navigate to the [Azure Portal](https://portal.azure.com/) and log in using your credentials.
2. Click on **App registrations** to register a new application.
3. Navigate to the **Overview** tab. Make a note of the Client ID and Tenant ID.
4. Click on the **Certificates & secrets** tab and create a new client secret. Keep this secret handy.
5. Go to the **API permissions** tab.
   - Click on "Add permissions."
   - Choose "APIs my organization uses."
   - Search for and select "Office 365 Exchange Online."
   - Add the `full_access_as_app` application permission.
6. You can now use the Client ID, Tenant ID, and Client Secret you've noted to configure the Outlook connector.

Reply:
Yes, this looks clean! 👍